### ISSUE TITLE
Renderer: Is it possible to get a reference to transcluded content?

### ISSUE THREAD

**Issue by matanlurey** [angular/angular#2320]
E.g., I have the following component:

```dart
@Directive(selector: '[bar]')
class BarDirective {
  final ElementRef elementRef;
  final ProtoViewRef protoViewRef;

  BarDirective(this.elementRef, this.protoViewRef);
}

@Component(selector: 'foo')
@View(
  template: r'''
    <template bar>PRE<content></content>POST</template>
  '''
)
class FooComponent {}
```

If I grab `bar`'s elementRef/protoViewRef, and add it to a view container elsewhere it only contains:

`PREPOST`

Is that working as intended?

@tbosch

**Comment by matanlurey** [comment]
Talked to tbosch@, because the native shadow DOM would not support this, neither does Angular's emulated layer. It may be possible to do something like this in the future as Angular evaluates alternatives. See also #1931.

**Comment by naomiblack** [comment]
@tbosch is P3 correct? Should this be P4?

**Comment by tbosch** [comment]
This will be solved by #2529

**Comment by tbosch** [comment]
This is solved with the new content projection. I also added test to ensure this here: https://github.com/angular/angular/blob/master/modules/angular2/test/core/compiler/projection_integration_spec.ts#L307
